src/sif/transitcost.cc

```diff
@@ -537,7 +537,7 @@ bool TransitCost::Allowed(const baldr::DirectedEdge* edge,
                           const baldr::GraphId& edgeid,
                           const uint64_t current_time,
                           const uint32_t tz_index,
-                          bool& has_time_restrictions) const {
+                          int& restriction_idx) const {
   if (flow_mask_ & kCurrentFlowMask) {
```

Comment:
this check doesnt belong here. transit doesnt care about traffic. we should delete this but its not related to your pr